src/crab_verifier.cpp

```diff
@@ -116,8 +116,14 @@ static checks_db generate_report(cfg_t& cfg,
 static void print_report(std::ostream& os, const checks_db& db, const InstructionSeq& prog) {
     os << "\n";
     for (auto [label, messages] : db.m_db) {
-        for (const auto& msg : messages)
+        for (const auto& msg : messages) {
+            if (label.line_info.has_value()) {
+                auto& [file, source, line, colum] = label.line_info.value();
```

Comment:
```suggestion
                auto& [file, source, line, _] = label.line_info.value();
```
avoid typo (column).  Ok to use `_` since column is unused below.